doc/ProfilesJSONDocumentation.md

```diff
@@ -0,0 +1,39 @@
+| **profiles** | **Array[Object]** | **Properties listed below are specific to each unique profile.** |
+| acrylicOpacity | Integer | Sets the transparency of the window for the profile. Accepts values from 0-1. |
+| background | String | Sets the background color of the profile. Overrides `background` set in color scheme if `colorscheme` is set. |
+| closeOnExit | Boolean | When set to `true`, the selected tab closes when `exit` is typed. When set to `false`, the tab will remain open when `exit` is typed. |
+| colorscheme | String | Name of the color scheme used in the profile that is defined under `schemes`. |
+| colorTable | Array[String] | Array of colors used in the profile if `colorscheme` is not set. Ordering is as follows: `[black, red, green, yellow, blue, magenta, cyan, white, bright black, bright red, bright green, bright yellow, bright blue, bright magenta, bright cyan, bright white]` |
+| commandline | String | Executable used in the profile. |
+| cursorColor | String | Sets the cursor color for the profile. |
+| cursorHeight | Integer | Sets the height of the cursor. Only works when `cursorShape` is set to `"vintage"`. Accepts values from 25-100. |
+| cursorShape | String | Sets the cursor shape for the profile. Possible values: `"vintage"`, `"bar"`, `"underscore"`, `"filledBox"`, `"emptyBox"` |
+| fontFace | String | Name of the font face used in the profile. |
+| fontSize | Integer | Sets the font size. |
+| foreground | String | Sets the foreground color of the profile. Overrides `foreground` set in color scheme if `colorscheme` is set. |
+| guid | String | Unique identifier of the profile. |
+| historySize | Integer | The number of lines above the ones displayed in the window you can scroll back to. |
+| icon | String | Image file location of the icon used in the profile. Displays within the tab and the dropdown menu. |
+| name | String | Name of the profile. Displays in the dropdown menu. |
+| padding | String | Sets the padding around the text within the window. Can have three different formats: `"#"` sets the same padding for all sides, `"#, #"` sets the same padding for top-bottom and right-left, and `"#, #, #, #"` sets the padding individually for top, right, bottom, and left. |
+| scrollbarState | String | Defines the visibility of the scrollbar. Possible values: `"visible"`, `"hidden"` |
+| snapOnInput | Boolean | When set to `true`, the window will scroll to the command input line when typing. When set to `false`, the window will not scroll when you start typing. |
+| startingDirectory | String | The directory the shell starts in when it is loaded. |
+| useAcrylic | Boolean | When set to `true`, the window will have an acrylic background. When set to `false`, the window will have a plain, untextured background. |
+| **schemes** | **Array[Object]** | **Properties listed below are specific to each color theme.** |

```

Comment:
```suggestion
| **schemes** | **Array[Object]** | **Properties listed below are specific to each color scheme.** |
```
I want to make sure we keep `schemes` separate from `themes`, because when we do add support for theming, it's going to be a very different thing than just the color schemes :)